### DISCUSSION TITLE
The chromedriver link blows up

### DISCUSSION THREAD

**Discussion by seunghoon4176** [seleniumbase/SeleniumBase#2665] [General]
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/123.0.6312.105/win64/chromedriver-win64.zip
This link blows up..
I can see Download Complete! then got some erros.. i cant use chromedriver..

**Comment by limhasic** [comment]
Your overtime duty is confirmed.

**Comment by mdmintz** [comment]
The Chromium Team changed the download location of drivers, and SeleniumBase was updated to use the newer location over a month ago in https://github.com/seleniumbase/SeleniumBase/issues/2519

Upgrade to a newer version of SeleniumBase.

**Reply by seunghoon4176** [reply]
My problem was using seleniumbase 4.21.4

**Reply by seunghoon4176** [reply]
But i updated my seleniumbase version, i saw same error

**Reply by mdmintz** [reply]
Use this code to check your actual version:

```python
from seleniumbase import __version__
print(__version__)
4.25.2
```

There are no more references to the old URLs anywhere in the code (`edgedl`)

<img width="1325" alt="Screenshot 2024-04-05 at 8 25 03 PM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/31131a69-fdbc-4043-954a-abbeb10f2cec">

**Comment by seunghoon4176** [comment]
pip show seleniumbase
Name: seleniumbase
Version: 4.25.2
Summary: A complete web automation framework for end-to-end testing.
Home-page: https://github.com/seleniumbase/SeleniumBase
Author: Michael Mintz
Author-email: mdmintz@gmail.com
License: MIT
Location: C:\Anaconda3\Lib\site-packages
Requires: attrs, beautifulsoup4, behave, certifi, chardet, charset-normalizer, colorama, cssselect, exceptiongroup, execnet, fasteners, filelock, h11, idna, iniconfig, markdown-it-py, mdurl, outcome, packaging, parameterized, parse, parse-type, pdbp, pip, platformdirs, pluggy, py, pygments, pynose, pyotp, pyreadline3, pytest, pytest-html, pytest-metadata, pytest-ordering, pytest-rerunfailures, pytest-xdist, pyyaml, requests, rich, sbvirtualdisplay, selenium, setuptools, six, sniffio, sortedcontainers, soupsieve, tabcompleter, trio, trio-websocket, typing-extensions, urllib3, wheel, wsproto

idk whi i saw this error always

**Reply by mdmintz** [reply]
Like I said, use this code to check your actual version:

```python
from seleniumbase import __version__
print(__version__)
4.25.2
```

If you have multiple installations of SeleniumBase on your system, you could be using the wrong version. The latest version definitely doesn't have `edgedl` anywhere in the code.

**Reply by seunghoon4176** [reply]
i am confusing now..

**Reply by seunghoon4176** [reply]
There's only newest version

**Comment by seunghoon4176** [comment]
It can caused by undetected driver?

**Reply by mdmintz** [reply]
There's no SeleniumBase code that points anywhere to `https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/`. You need to show a full stack trace.

`sbase get uc_driver` will download the correct `driver` on recent versions.

**Reply by seunghoon4176** [reply]
i deleted python and deleted all modules. i thought i using anaconda, but old python312 folder has seleniumbase 4.19.  thx!
